pkg/conversions/role.go

```diff
@@ -524,6 +543,9 @@ func RoleFromResourcePermissions(rp *provider.ResourcePermissions, islink bool)
 		}
 	}
 	if r.ocsPermissions == PermissionCreate {
+		if rp.GetPath && rp.InitiateFileDownload && rp.ListContainer && rp.Move {
+			r.Name = RoleEditorLite
+		}
```

Comment:
```suggestion
		if rp.GetPath && rp.InitiateFileDownload && rp.ListContainer && rp.Move {
			r.Name = RoleEditorLite
			return r
		}
```
Otherwise `r.Name` is overwritten in the line below.

Reply:
hm ... I don't see a test covering RoleFromResourcePermissions ... should we add them?

Reply:
Feel free to add as many unit tests as you like :smiley:

Reply:
`RoleFromResourcePermissions` in general feels like it could use some overhaul. Why doing this via settting the ocsPermission first an then "guess" the role name from that. That procedure seems somewhat lossy.

I don't dare touching this currently. But we should re-work that thing once we can get rid of the ocsPermission stuff.